constants/constants.go

```diff
@@ -44,3 +45,14 @@ const TokenExpiryDurationRegister = time.Hour * 24 * 7          // 1 week
 const TokenExpiryDurationLogin = time.Hour * 24 * 30            // 30 days
 const TokenExpiryDurationDefault = time.Hour * 24 * 30          // 30 days
 const TokenExpiryDurationPermanent = time.Hour * 24 * 365 * 100 // 100 years
+
+var DealMaxPrice abi.TokenAmount
+var VerifiedDealMaxPrice = abi.NewTokenAmount(0)
+
+func init() {
```

Comment:
Why did we need to move this init() to constants?

Reply:
no particular reason besides colocating default values (given all it does is set the default price)

Reply:
Ok I see that you're trying to initialize the value of the dealmax price so this change seems reasonable